src/plugins/mf_priority.cpp

```diff
@@ -96,11 +96,12 @@ static void rec_update_cb (flux_t *h,
                            const flux_msg_t *msg,
                            void *arg)
 {
-    char *uid, *fshare, *bank;
+    char *uid, *fshare, *bank, *default_bank;
 
-    if (flux_request_unpack (msg, NULL, "{s:s, s:s, s:s}",
+    if (flux_request_unpack (msg, NULL, "{s:s, s:s, s:s, s:s}",
```

Comment:
Just an idea/question: It might be slightly awkward here that the protocol requires that the `default_bank` is sent along with every `bank` update. Unless I've missed something below, if `default_bank` != `bank`, then `default_bank` is unused. 

Another idea would be to have `default_bank` be a boolean, but I'm not sure if this would make the client side more difficult.

Also, I realize you have an issue open to turn this simpler protocol into a "bulk update" protocol, so that you can update multiple users in one RPC. If that is the case, then maybe a cleanup of the RPC design could be done at that time. No reason to fix the current design if it works for now.

Reply:
Ok, actually I see now above that this protocol simplifies the current bulk_update.py client, so it is probably fine to leave as is for now.